draft-ietf-taps-impl.md

```diff
@@ -1146,7 +1146,7 @@ This document has no actions for IANA.
 
 ## Considerations for Candidate Gathering
 
-Implementations should avoid downgrade attacks that allow network interference to cause the implementation to select less secure, or entirely insecure, combinations of paths and protocols.
+As described in the Security Considerations of {{I-D.ietf-taps-arch}}, TAPS forbids applications from specifying protocol stacks with inequivalent security properties for gathering and racing. Implementations must avoid downgrade attacks that allow network interference to cause the implementation to select less secure, or entirely insecure, combinations of paths and protocols.
```

Comment:
```suggestion
The Security Considerations of the TAPS Architecture {{I-D.ietf-taps-arch}} forbids gathering and racing with protocol stacks that do not have equivalent security properties. Therefore, implementations need to avoid downgrade attacks that allow network interference to cause the implementation to select less secure, or entirely insecure, combinations of paths and protocols.
```